test/bdd/fixtures/scripts/vcs_configure.sh

```diff
@@ -128,3 +128,23 @@ do
    n=$((n+1))
    sleep 5
 done
+
+n=0
+maxAttempts=30
+until [ $n -ge $maxAttempts ]
+do
+   responseCreatedTime=$(curl --header "Content-Type: application/json" \
+   --request POST \
+   --data '{"name":"vc-issuer-p256", "uri":"http://vc-issuer-p256.com", "signatureType":"JsonWebSignature2020", "signatureRepresentation":1,"uniRegistrar":{"driverURL":"https://uni-registrar-web.trustbloc.local/1.0/register?driver-did-method-rest"}},"didKeyType":"P256"' \
```

Comment:
also need an interop version that disable credential status check.

Reply:
[Added](https://github.com/trustbloc/edge-sandbox/pull/241/files#diff-5e6d8e21c3faa9d1d996901b1ae16f25R158).

Reply:
Note - VCs created with interop profiles fail - https://github.com/trustbloc/edge-sandbox/issues/242.